### PR TITLE
fix: rn-start

### PR DESCRIPTION
**Pull request by helsonxiao**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

- 修复直接 kill 8081 端口可能会错误 kill 其它进程的问题（比如 Surge
- 补上 `shell-utils`

### CODE REVIEW

**Comment by helsonxiao** [comment]
CI 因为 storage 没有 mock 掉所以挂了，可以无视，另一个 pr 已经加上了 mock 配置

File: scripts/start.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const exec = require('shell-utils').exec;
 
 
 function run() {
-  exec.killPort(8081);
+  exec.execSync('lsof -i:8081 | grep \'node\' | awk \'{print $2}\' | xargs kill -9');
```
Comment (bang88) [inline scripts/start.js:5]: 会不会 有人在windows下面运行。

Comment (helsonxiao) [reply scripts/start.js:5]: 尴尬，查了下这几个都不支持 windows，有什么准确一些的做好兼容的 kill api 吗

Comment (helsonxiao) [reply scripts/start.js:5]:
```bash
COMMAND   PID       USER   FD   TYPE             DEVICE SIZE/OFF NODE NAME
Surge   82466 helsonxiao   55u  IPv4 0x8dfeb8b7e0c832a1      0t0  TCP hac2:60770->11.11.11.11:sunproxyadmin (ESTABLISHED)
Surge   82466 helsonxiao  139u  IPv4 0x8dfeb8b7e0c77ee1      0t0  TCP hac2:60331->11.11.11.11:sunproxyadmin (ESTABLISHED)
node    86809 helsonxiao   24u  IPv6 0x8dfeb8b7d8c996f1      0t0  TCP *:sunproxyadmin (LISTEN)
```
这是我在 mac 上执行 `lsof -i:8081` 的结果

Comment (erichua23) [reply scripts/start.js:5]: 看了一下上面的讨论，这个修改也没准确识别到是杀掉老的packager，所以还是存在误杀可能的。只是误杀概率变小了，所以感觉这样修的意义不大。

或者改成不要在这里杀8081，或者改成都杀，其实更容易理解一点，杀之前增加一些warnning告知使用者就好。

**Pull request closed without merging**